client/components/MainDescriptionComponents/MainDescription.jsx

```diff
@@ -1,4 +1,15 @@
 import React from 'react';
+import styled from 'styled-components';
+
+const DescriptionContainer = styled.div`
+  padding-top: 15px;
+  width: 33%;
+  overflow-wrap: break-word;
```

Comment:
oh interested in what overflow does

Reply:
It will prevent words from breaking out of your container. The words will break into a new line!